docs/src/pages/[platform]/getting-started/migration/migration.angular.mdx

```diff
@@ -25,29 +25,8 @@ yarn add aws-amplify @aws-amplify/ui-angular@3.x
 
 The following breaking changes were introduced in `@aws-amplify/ui-angular` major version `3.0`:
 
-
-#### 1. Update the handleSignUp function override call
```

Comment:
Reordered this item to below because `handleSignUp` is not as common of use case.